docs/versioned_docs/version-v3.2.0/02-Features/publishing-events.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Publishing Events"
+layout: default
+---
+
+## Publishing Events
+
+![](https://img.shields.io/badge/Available%20starting-v3.0-green?link=https://github.com/arcus-azure/arcus.eventgrid/releases/tag/v3.0.0)
+
+We provide support for publishing custom events to a custom Azure Event Grid Topics.
+
+Import the following namespace into your project:
+
+```csharp
+using Arcus.EventGrid.Publishing;
+```
+
+Next, create an `EventGridPublisher` instance via the `EventGridPublisherBuilder` which requires the endpoint & authentication key of your custom topic endpoint.
+
+```csharp
+var eventGridPublisher = EventGridPublisherBuilder
+                                .ForTopic(topicEndpoint)
+                                .UsingAuthenticationKey(endpointKey)
+                                .Build();
+```
+**Publishing EventGridEvent's**
+
+Create your event that you want to publish
+
+```csharp
+string licensePlate = "1-TOM-337";
```

Comment:
@tomkerkhove 's legacy is still present here :)

Reply:
Yeah, hihi 😄 .